code/game/asteroid.dm

```diff
@@ -173,7 +174,7 @@ var/global/list/spawned_surprises = list()
 			y_size = 5
 			areapoints = 25
 
-	if(!monolith)
+	if(monolith != 1)

```

Comment:
!monolith is the same as monolith == 0 and monolith == null in scripting languages, which is neater